myuw/static/js/ws_data.js

```diff
@@ -224,6 +224,7 @@ WSData = {
             var minutes_till_deadline = grading_deadline.diff(ref, 'minutes');
             var deadline_in_24_hours = (minutes_till_deadline >= 0 &&
                                         minutes_till_deadline <= (24 * 60));
+            var comparison_date = moment(new Date(window.card_display_dates.comparison_date));
```

Comment:
I think for these moment calls you should have moment parse the string rather than Date.